src/Clide.Interfaces/Extensions/ServiceProviderExtensions.cs

```diff
@@ -160,22 +179,26 @@ public static TPackage GetLoadedPackage<TPackage>(this IServiceProvider serviceP
     /// automatically if needed.
     /// </summary>
     /// <returns>The fully loaded and initialized package.</returns>
-    public static IServiceProvider GetLoadedPackage(this IServiceProvider serviceProvider, Guid packageId)
+    public static System.Threading.Tasks.Task<IServiceProvider> GetLoadedPackageAsync(this IServiceProvider serviceProvider, Guid packageId)
     {
         var jtf = GetJTF(serviceProvider);
 
-        return jtf.Run(async () =>
+        return jtf.RunAsync(async () =>
```

Comment:
Why don't converting the method to an async method and get rid of the JTF.RunAsync and instead awaiting directly?